src/styles/blocks/search.css

```diff
@@ -0,0 +1,50 @@
+.search {
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  background-image: url('../../images/Search.svg');
+  background-repeat: no-repeat;
+  font-family: Avenir, serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't load. For example, 'Avenir, Helvetica, Arial, sans-serif'. This ensures text is still legible with a default system font if the custom font fails to load.